tensorhive/controllers/schedule/create_schedule_controller.py

```diff
@@ -0,0 +1,40 @@
+from tensorhive.models.RestrictionSchedule import RestrictionSchedule
+from datetime import datetime
+from tensorhive.utils.Weekday import Weekday
+from tensorhive.authorization import admin_required
+from tensorhive.config import API
+S = API.RESPONSES['schedule']
+G = API.RESPONSES['general']
+
+
+@admin_required
+def create(schedule):
+    try:
+        days = []
+        for day in schedule['scheduleDays']:
+            days.append(Weekday[day])
```

Comment:
We can simplify this:
```suggestion
        days = [Weekday[day] for day in schedule['scheduleDays']]
```